llmfoundry/utils/builders.py

```diff
@@ -133,6 +134,8 @@ def build_logger(name: str, kwargs: Dict[str, Any]) -> LoggerDestination:
         return MLFlowLogger(**kwargs)
     elif name == 'inmemory':
         return InMemoryLogger(**kwargs)
+    elif name == 's3':
```

Comment:
what is going on here?

Reply:
I need to add an s3 logger: 
```
loggers:
  s3: {bucket_uri: s3://mosaicml-internal-checkpoints-shared/ }

```

Reply:
https://github.com/mosaicml/composer/blob/e32ac0f460d8092681bae622fdcf21197b586d95/composer/profiler/torch_profiler.py#L129-L131

Reply:
hmm this does not match the way we support things these days which is that if you pass in a remote path, we automatically whatever objects are needed for you. also the way you've implemented it, it only works for s3, which is not ideal. i'd suggest refactoring composer to check if the trace path is remote and if so create the RUD

Reply:
removed